Antd trends updated layout

EDsCODE opened the pull request:
## Changes

- finishes #532 

## Checklist
~~- [ ] All querysets/queries filter by Team~~
- [ ] Code reviewed
- [ ] QA'd

EDsCODE committed:
initial reformat

EDsCODE committed:
minor adjustments

EDsCODE committed:
screen size adjustments

EDsCODE commented on frontend/src/lib/hooks/useWindowSize.js:
```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react'
+
+export function useWindowSize() {
```
REF: https://usehooks.com/useWindowSize/

Could use react-responsive but refrained from adding more packages atm

EDsCODE committed:
fix breakdown button

EDsCODE committed:
fix breakdown button

EDsCODE committed:
session filter width

timgl reviewed (commented):
Very nice!

The Card bar isn't aligned correctly. 
![image](https://user-images.githubusercontent.com/1727427/80082080-c1e0d480-854b-11ea-89fa-f0fbee17dac9.png)


The whole layout looks a little... off on a larger screen like mine. I wonder if the solution is maybe to remove the `min-height: 85vh`

![image](https://user-images.githubusercontent.com/1727427/80082601-7ed33100-854c-11ea-953a-25412ac92ad2.png)

To this:
![image](https://user-images.githubusercontent.com/1727427/80082715-a75b2b00-854c-11ea-844b-9ab5c300e2aa.png)

EDsCODE committed:
adjust sizing

EDsCODE commented:
the minheight is meant to make the cards the same height for consistency purposes. We could let the items within the cards determine the height too but I just thought the consistent heights looked a bit more put-together. I changed the condition to display specifically when the window is longer horizontally than vertically so on long vertical windows it should switch back to our original layout!

and adjusted the padding for the card bar (I was fooling with it because I wanted the bottom of that grey container to align with the separator under the tabs on the settings card)

timgl commented:
I had a quick play around with this. I think the actual heuristic that makes more sense is that if the height of the screen is > 1200, it should _not_ have the min-height. That way if the screen is wide as well as tall it will still be collapsed.

EDsCODE committed:
removed minheight

EDsCODE commented:
Removed minheight! Card heights will be sized by content inside so there shouldn't be elongated columns
